orttraining/orttraining/python/training/ortmodule/_custom_op_symbolic_registry.py

```diff
@@ -78,6 +78,12 @@ def diagonal(g, self, offset, dim1, dim2):
     return g.op("com.microsoft::ATenOp", self, offset, dim1, dim2,
                 name_s='aten::diagonal')
 
+@register_symbolic('multinomial')
+def multinomial(g, self, num_samples, replacement=False, generator=None):
+    if generator is not None and not sym_help._is_none(generator):
```

Comment:
should the condition be 'and' or 'or'

Reply:
this is following https://github.com/pytorch/pytorch/blob/bbe25af0df2f2f6a58f6cc4ee5899efcba2ee336/torch/onnx/symbolic_opset9.py#L2879